README.md

```diff
@@ -12,7 +12,7 @@ Decentralized, open-source (MIT), C/C++ package manager.
 - Homepage: https://conan.io/
 - Github: https://github.com/conan-io/conan
 - Docs: https://docs.conan.io
-- Slack: https://cpplang.slack.com (#conan channel)
+- Slack: https://cpplang.slack.com (#conan channel. See how to join below)
```

Comment:
Can't find below any details, maybe add here the link to the invite too?

Reply:
I just updated now with the invitation link from cppalliance website. I guess it's much easier now. The "below" section was actually the part that explains about Conan for C++ Devs but also points the alliance website link.